torch/onnx/symbolic_opset9.py

```diff
@@ -6629,3 +6631,35 @@ def onnx_placeholder(g: jit_utils.GraphContext, *inputs, **attrs):
     env = g.env
 
     return torch._C._jit_onnx_convert_pattern_from_subblock(block, node, env)
+
+
+@_onnx_symbolic("aten::resolve_conj")
+@_onnx_symbolic("aten::resolve_neg")
+@_beartype.beartype
+def _unsupported_complex_operators_noop(g: jit_utils.GraphContext, input: _C.Value):
```

Comment:
naming: nit: since there isn't really anything unsupported, maybe just call it `_noop`?